README.md

```diff
@@ -19,14 +19,14 @@ The chart is published to public Helm repository, [hosted on GitHub itself](http
 Add Helm repository
 
 ```
-helm repo add opal https://permitio.github.io/opal-helm-chart
+helm repo add permitio https://permitio.github.io/opal-helm-chart
```

Comment:
Why so ? don't we want to decouple opal from permit ?

Reply:
Hmmm...
Opal is the chart and permitio is the repo, maybe we'll have other charts?
We do add the link "https://permitio.github.io/opal-helm-chart" so you're right it's scoped to opal only.
But my association was the docker images `permitio/opal-server` etc...